packages/klighd-core/src/sidebar/sidebar.tsx

```diff
@@ -110,12 +118,28 @@ export class Sidebar extends AbstractUIExtension {
         // Show or hide the panel
         if (currentPanel) {
             this.containerElement.classList.add('sidebar--open')
+            // Set width of sidebar to maximum width of all opened panels.
+            this.maxWidth = Math.max(this.maxWidth, this.containerElement.clientWidth)
```

Comment:
This line (and its copy in the else case) can be pulled in front of the if statement to avoid duplicate code